Dockerfile

```diff
@@ -1,4 +1,5 @@
 ARG BASE_IMAGE_FULL
+ARG BASE_IMAGE_MINIMAL
```

Comment:
Sorry, one nit 😬  could you move this to the first patch where it really belongs to?

Reply:
yup, done